src/torrent/peer/client_info.h

```diff
@@ -106,7 +108,7 @@ class LIBTORRENT_EXPORT ClientInfo {
 
   // We don't really care about cleaning up this as deleting an entry
   // form ClientList shouldn't happen.
-  info_type*          m_info;
+  std::optional<info_type> m_info;
```

Comment:
Optional has added cost of a bool, and isn't necessary here as we should never have a null value.

In fact we can remove the m_info completely and just store the `const char*`, which all get initialized from static DATA.

Reply:
Adjusted.